src/main/java/org/mockito/internal/stubbing/defaultanswers/ReturnsEmptyValues.java

```diff
@@ -109,8 +112,20 @@ Object returnValueFor(Class<?> type) {
             return new LinkedHashMap<Object, Object>();
         } else if ("java.util.Optional".equals(type.getName())) {
             return JavaEightUtil.emptyOptional();
+        } else if ("java.util.OptionalDouble".equals(type.getName())) {
```

Comment:
Well my reasoning was fast : 

1. We forgot that in mockito 2, as a user I usually see that as a bug.
2. The change is consistent with the goal of mockito 2 aka support java 8 best way.
3. The change is breaking for users that uses primitive variant and that expect null for these stubs, which is not what those type are made for, so pretty rare.

All in a all this a reasonable change, that imo does not mean mockito should bump any version or postpone the change.